porogo/frontend/static/styles/views/_home.scss

```diff
@@ -1,44 +1,38 @@
 
 @import '../common/variables';
 
-.top-content {
-    height: $body-height;
```

Comment:
I had this so that the whole welcome screen could be seen with the header on screen. Do we want to see them both at once or have the bottom of the welcome screen below the fold

Reply:
Yeah it'd probably be better if we could both at the same time.

Reply:
I put it back.